src/comp_loinc/analysis/app.py

```diff
@@ -0,0 +1,193 @@
+"""Interactive visualization app"""
```

Comment:
File which you can run locally by running this file using `python app.py`. There's also a `makefile` goal for it at the bottom of the makefile.

Deployed to: https://comp-loinc.onrender.com/

Reply:
Static TSV files are committed so that the app doesn't have to run this code prior to launching.

They are updated automatically during `make all`.